packages/web-runtime/src/components/Account/CalDavUrl.vue

```diff
@@ -0,0 +1,132 @@
+          )
+        }}</oc-table-td>
+      </oc-table-tr>
+      <oc-table-tr class="account-page-info-caldav-url">
+        <oc-table-td>{{ $gettext('CalDAV URL') }}</oc-table-td>
+        <oc-table-td colspan="2">
+          <div class="oc-flex oc-flex-middle">
+            <span class="oc-text-truncate">{{ calDavUrl }}</span>
+            <oc-button
+              v-oc-tooltip="$gettext('Copy CalDAV URL')"
+              class="oc-ml-m"
+              appearance="raw"
+              data-testid="copy-caldav-url"
+              size="small"
+              :aria-label="$gettext('Copy CalDAV URL to clipboard')"
+              no-hover
+              @click="copyCalDavUrlToClipboard"
+            >
+              <oc-icon :name="copyCalDavUrlIcon" size="small" />
+              <span class="oc-ml-2xs">{{ $gettext('Copy') }}</span>
+            </oc-button>
+          </div>
+        </oc-table-td>
+      </oc-table-tr>
+      <oc-table-tr class="account-page-info-caldav-username">
+        <oc-table-td>{{ $gettext('Username') }}</oc-table-td>
+        <oc-table-td colspan="2">
+          <div class="oc-flex oc-flex-middle">
+            <span>{{ user.onPremisesSamAccountName }}</span>
+            <oc-button
+              v-oc-tooltip="$gettext('Copy CalDAV username')"
+              class="oc-ml-m"
+              appearance="raw"
+              data-testid="copy-caldav-username"
+              size="small"
+              :aria-label="$gettext('Copy CalDAV username to clipboard')"
+              no-hover
+              @click="copyCalDavUsernameToClipboard"
+            >
+              <oc-icon :name="copyCalDavUsernameIcon" size="small" />
+              <span class="oc-ml-2xs">{{ $gettext('Copy') }}</span>
+            </oc-button>
+          </div>
+        </oc-table-td>
+      </oc-table-tr>
+      <oc-table-tr class="account-page-info-caldav-password">
+        <oc-table-td>{{ $gettext('Password') }}</oc-table-td>
+        <oc-table-td colspan="2">
+          {{ $gettext('An app token needs to be generated and then can be used.') }}
+        </oc-table-td>
+      </oc-table-tr>
+    </account-table>
+  </div>
+</template>
+
+<script setup lang="ts">
+import { ref, computed, onMounted, unref } from 'vue'
+import { storeToRefs } from 'pinia'
+import { useClientService, useConfigStore, useUserStore } from '@opencloud-eu/web-pkg'
+import { useGettext } from 'vue3-gettext'
+import AccountTable from './AccountTable.vue'
+
+const { $gettext } = useGettext()
+const userStore = useUserStore()
+const { user } = storeToRefs(userStore)
+const configStore = useConfigStore()
+const clientService = useClientService()
+
+const calDavAvailable = ref(false)
+const copiedIcon = 'check'
+const copyIcon = 'file-copy'
+const checked = ref(false)
+
+const calDavUrl = computed(() => {
+  return `${configStore.serverUrl}`
+})
```

Comment:
This doesn't need to be a `computed`. Computed properties are only necessary when values as expected to change, which is not the case here. You could just use `configStore.serverUrl` directly where needed.